src/Nethermind/Nethermind.Runner/Startup.cs

```diff
@@ -40,7 +40,9 @@ public class Startup
             ContractResolver = new CamelCasePropertyNamesContractResolver()
         };
         
-        private IJsonSerializer _jsonSerializer = new EthereumJsonSerializer();
+        private IJsonSerializer _jsonSerializer = CreateJsonSerializer();
+
+        private static EthereumJsonSerializer CreateJsonSerializer() => new EthereumJsonSerializer(NullValueHandling.Include);
```

Comment:
this cannot be this way - there are plenty of places where they complain about nulls - it has to be done specifically on some properties only